src/app/chat/event_handling.nim

```diff
@@ -67,6 +67,10 @@ proc handleChatEvents(self: ChatController) =
   self.status.events.on("chat:connected") do(e: Args):
     self.view.setConnected(true)
 
+  self.status.events.on(PendingTransactionType.BuyingStickerPack.event) do(e: Args):
+    var data = TransactionMinedArgs(e).data
+    self.view.installStickerPack(data.parseInt)
```

Comment:
If we update the `TransactionMinedArgs` as per comment below, then we could change this to look like:

```suggestion
    let tx = TransactionMinedArgs(e)
    if tx.success:
      self.view.installStickerPack(tx.data.parseInt)
    else:
      # self.view.toastMessage(message = tx.revertReason, error = true) # for when the toast messages in the design are implemented
```